packages/account/src/Styles/account.scss

```diff
@@ -764,6 +764,12 @@ $MIN_HEIGHT_FLOATING: calc(
             width: 100%;
             height: 100%;
 
+            @media (min-width: 1025px) and (max-width: 1199px) {
```

Comment:
why not max-width only?, what happen on 1024 and below?

Reply:
1024px to 1200 is still desktop and there are 3 columns so as we discussed we make the article width smaller to make the Fields wider

Reply:
Below 1024px is the Tablet screen size where grid turns to have only a single column